Src/AutoFixture/BooleanSwitch.cs

```diff
@@ -27,6 +27,7 @@ public BooleanSwitch()
         /// <see langword="true"/>, followed by <see langword="false"/> at the next invocation, and
         /// so on.
         /// </returns>
+        [Obsolete("Please use the Create(request, context) method as this overload will be removed to make API uniform.")]
```

Comment:
Perhaps, not a blocker, but `Create(request, context)` target only the C# developers. What about those using F# or VB.NET, etc?

Perhaps, we could reword this as *the Create method taking a request and a context as parameters*.

Reply:
And what is the issue with F# and those overload? Why it isn't suitable for F#?

Reply:
We have a lot of `ISpecimenBuilder` implementations in our project and all them contain a single `Create` method overload. Those ones in the PR due to some reasons contain extra overloads. My point is that we should unify them all. If that overload is not compatible with F# or VB, we should revisit that at the whole project level, rather than fixing it for these particular builders only.

However, I still don't understand what are the issues with compatibility..

Reply:
Well, `Create(request, context)` becomes `Create (request, context)` in F# (but both ways will actually compile fine) though I don't know how it looks for VB.NET. I guess, it's fine as-is.

Reply:
Well, that will indeed be a common problem for all the builders we have (and likely most of the fixture API). Therefore if you work with F#, it's better to create a set of own helpers to get benefits from F# syntax.